src/app/containers/Timestamp/index.jsx

```diff
@@ -1,20 +1,30 @@
 import React from 'react';
 import { number } from 'prop-types';
+import moment from 'moment-timezone';
 import Timestamp from '../../components/Timestamp';
 import relativeTime from './relativeTimestamp';
 import {
   longNumeric,
   shortAlphaNumeric,
+  alphaNumericDatetime,
   isValidDateTime,
   formatUnixTimestamp,
   isTenHoursAgoOrLess,
 } from './timestampUtilities';
 import { GridItemConstrainedMedium } from '../../lib/styledGrid';
 
+const isToday = timestamp => {
+  const today = moment(Date.now());
+  return today.isSame(timestamp, 'day');
+};
+
+const formatType = timestamp =>
+  isToday(timestamp) ? alphaNumericDatetime : shortAlphaNumeric;
```

Comment:
Is it just me that struggles to read these names and know what the format will be?

We could rename:

- `alphaNumericDateTime` => `dateAndTime`
- `shortAlphaNumeric` => `dateOnly`

Naming is the hardest thing in computer science. These suggestions are just an opinion! What do you think?

Reply:
I agree 👍 I really didn't like the naming and I think those suggestions work.